.github/workflows/crux-llvm-build.yml

```diff
@@ -186,9 +192,16 @@ jobs:
           CLANG: "/c/Program Files/LLVM/bin/clang"
 
       - shell: bash
+        name: Build crux-llvm SVComp tool
         run: .github/ci.sh build exe:crux-llvm-svcomp
         if: runner.os != 'Windows'
 
+      - shell: bash
+        name: Build crucible wasm support library
+        run: |
+          (cd dependencies/haskell-wasm; nix-shell -p hpack --run hpack)
+          ./github/ci.sh build crucible-wasm
```

Comment:
```suggestion
          ./.github/ci.sh build crucible-wasm
```

I think?

Reply:
Actually, I just need to remove the slash, but good catch.  The moral of the story is one shouldn't code while distracted.